### PR TITLE
New example contribution for WVD

### PR DESCRIPTION
**Pull request by fberson**
New example contribution for WVD

### CODE REVIEW

**Comment by fberson** [comment]
@rjmholt, I contributed a example but the test fails, I'm not sure why, maybe you can help?

**Comment by rjmholt** [comment]
@fberson thanks for contributing this! The test fails because the template still has the PSVersion and template hash in it -- if you remove those it should work. The PSVersion changes across PowerShell versions, so has to be taken out for tests in PS 5.1 and PS 7 to pass. I need to rewrite the tests to ignore those, but for now just removing those properties should do.

In the test failure message you should see this as:

```
   Expected $true, because object should contain property 'psarm-psversion' at '/.metadata._generator', but got $false.
```

**Comment by iSazonov** [comment]
@rjmholt Could you please enable anonymous access to Azure pipelines so that we can see CI process and results as in PowerShell repo?

**Comment by fberson** [comment]
@rjmholt, thanks! That fixed it! Ready for approval :)

**Comment by rjmholt** [comment]
@iSazonov @fberson are you able to see CI now?

**Comment by rjmholt** [comment]
Looks like I can without logging in 🙂

File: examples/wvd-backplane/wvd-backplane.parameters.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "hostpoolName": "myFirstHostpool",
+    "appgroupName": "myFirstAppGroup",
+    "workspaceName": "myFirstWorkspace"
+}
```
Comment (rjmholt) [inline examples/wvd-backplane/wvd-backplane.parameters.json:5]:
```suggestion
}

```

File: examples/wvd-backplane/wvd-backplane.psarm.ps1 [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  Resource $workspaceName -Namespace 'Microsoft.DesktopVirtualization' -Type 'workspaces' -ApiVersion '2019-12-10-preview' -Location $wvdbackplanelocation {
+    properties {
+      friendlyName $workspaceNameFriendlyName
+      applicationGroupReferences (resourceId 'Microsoft.DesktopVirtualization/applicationGroups' $appgroupName)
+    }
+    DependsOn @(
+      resourceId 'Microsoft.DesktopVirtualization/applicationGroups' $appgroupName
+    )
+  }
+}
```
Comment (rjmholt) [inline examples/wvd-backplane/wvd-backplane.psarm.ps1:71]:
```suggestion
}

```

**Review by rjmholt — APPROVED** [review]

**Pull request merged**